tools/add_triage_signature.py

```diff
@@ -48,33 +53,60 @@
 """
 
 
+def config_logger(verbose: bool):
+    global logger
```

Comment:
I thought since it is already declared as global, I should reach it like that.
But actually `logging` manager takes care of that and we can just reach it with `    logger = logging.getLogger("__main__")
`

Reply:
I thought since it is already declared as global, I should reach it like that.
But actually `logging` manager takes care of that and we can just reach it with `    logger = logging.getLogger("__main__")
`